### ISSUE TITLE
Question regards .57 when computing F1 score.

### ISSUE THREAD

**Issue by zshyang** [EdwardSmith1884/GEOMetrics#17]
Dear Edward Smith,

The code to compute f1 score 		
`dist_to_pred = torch.sqrt(torch.sum((.57*pred_counters - .57*gt_points.contiguous().view(-1,3))**2, dim  = 1 )).view(batch_size, -1)`
https://github.com/EdwardSmith1884/GEOMetrics/blob/09471d70f441482319543bb5214eaaab38f64507/utils.py#L490

I am not sure why .57 is needed?

**Comment by EdwardSmith1884** [comment]
This is the scaling everyone does for the 3Dr2n2 Dataset. When they made the images they scaled down the objects by this amount, so I do the same to make the object have the same size. You can find examples of this is other repos too.

**Comment by zshyang** [comment]
Thanks a lot!
